### PR TITLE
Feature/store locator

### PR DESCRIPTION
**Pull request by LeonardMeagher2**

### CODE REVIEW

File: src/LocationField/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+```js
```
Comment (ryanpcmcquen) [inline src/LocationField/README.md]: @LeonardMeagher2, aren't we using the component name for the docs (`LocationField.md`)?

File: docs/serve.config.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+import 'dotenv/config';
```
Comment (ryanpcmcquen) [inline docs/serve.config.js]: @LeonardMeagher2, I think this file moved to `builder/docs`, right?

File: src/StoreList/StoreList.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+```jsx
+<StoreProvider>
+	<StoreList sortBy={'distance'} limit={10} />
+</StoreProvider>
+```
```
Comment (ryanpcmcquen) [inline src/StoreList/StoreList.md]: @LeonardMeagher2, would you mind adding the Habitat code also?

File: src/StoreCard/StoreCard.md [hunk 4]
```diff
@@ -0,0 +1,8 @@
+```jsx
+let Typography = require('@material-ui/core/Typography').default;
+<StoreProvider>
+	<StoreCard storeId="001" detailed={true}>
+		<Typography> Children nodes land in this column</Typography>
+	</StoreCard>
+</StoreProvider>;
+```
```
Comment (ryanpcmcquen) [inline src/StoreCard/StoreCard.md]: @LeonardMeagher2, would you add the Habitat code here also?

File: docs/serve.config.js (hunk 2 above)
Comment (LeonardMeagher2) [reply docs/serve.config.js]: dotenv/config just looks for a .env and adds the variables in it the the process.env object.

File: src/StoreList/StoreList.md (hunk 3 above)
Comment (LeonardMeagher2) [reply src/StoreList/StoreList.md]: I'm not done with this component, and plan to add it after the component is done.

Comment (LeonardMeagher2) [reply src/StoreList/StoreList.md]: But Some of the functionality in this branch is needed for your component.

File: docs/serve.config.js (hunk 2 above)
Comment (ryanpcmcquen) [reply docs/serve.config.js]: @LeonardMeagher2, I just mean that this file is in the wrong place, nothing looks wrong with the specific file. I think this should live under `builder/docs/serve.config.js` (to not conflict with the static docs files).

File: src/StoreCard/StoreCard.md (hunk 4 above)
Comment (LeonardMeagher2) [reply src/StoreCard/StoreCard.md]: See my reply to the previous request.

File: src/LocationField/README.md (hunk 1 above)
Comment (LeonardMeagher2) [reply src/LocationField/README.md]: Fixed

File: docs/serve.config.js (hunk 2 above)
Comment (LeonardMeagher2) [reply docs/serve.config.js]: @ryanpcmcquen  Oh this file needs to be deleted

Comment (ryanpcmcquen) [reply docs/serve.config.js]: @LeonardMeagher2, yeah, unless this has changes for the current `builder/docs/serve.config.js`, I didn't diff them so I wasn't sure.

**Pull request merged**